### ISSUE TITLE
Bad UI of Recent Changes Page

### ISSUE THREAD

**Issue by Yashs911** [internetarchive/openlibrary#4415]
For tablet and mobile viewpoint:
### Evidence / Screenshot (if possible)
![image](https://user-images.githubusercontent.com/64412143/104537658-635f0780-5640-11eb-8753-05442ddab21d.png)

### Relevant url?

https://openlibrary.org/recentchanges

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Chrome
- **Operating system?** Windows 10
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

Solution:
Tablet:
![image](https://user-images.githubusercontent.com/64412143/104538036-21829100-5641-11eb-9c0f-79f9c5d450ce.png)
Mobile:
![image](https://user-images.githubusercontent.com/64412143/104538068-30694380-5641-11eb-9e80-68d72bf230ff.png)

### Stakeholders

@jdlrobson

**Comment by jdlrobson** [comment]
Without data on who's using these pages on a mobile phone I'm going to guess that not many are and this is low priority. Adding a width and some horizontal scrolling is probably an okay solution for now.

**Comment by arnab15** [comment]
I'm new to opensource I have knowledge of HTML,CSS,JS,Nodejs,Reactjs ,Mongodb,I want to contribute here how should I start.I'm unable to setup the dev environment.is anyone help me to setup dev env?

**Comment by Yashs911** [comment]
Hi, @arnab15 To set up the dev environment, follow these steps:
1) Fork and Clone Openlibrary repo, set up an upstream link
2) Download and Install Docker
3) After that, follow [Docker Instructions]( https://github.com/internetarchive/openlibrary/blob/master/docker/README.md) to set up the development environment.

and regarding this issue, I want to work on this.😄 
You can find other good first issues [here](https://github.com/internetarchive/openlibrary/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22)

**Comment by arnab15** [comment]
thank you @Yashs91  .It's very difficult to set up on windows 10😥.I

**Comment by Yashs911** [comment]
> thank you @YashS91 .It's very difficult to set up on windows 10😥.

@arnab15 No its not. I am also using windows 10 home edition

**Comment by arnab15** [comment]
@Yashs911 How much disk space it consumes in your pc?

**Comment by Yashs911** [comment]
@arnab15 15-20 GB. because of that my C drive was almost full so today I shifted the image location to D Drive

**Comment by Yashs911** [comment]
@arnab15, did you install docker?

**Comment by arnab15** [comment]
yes I have docker install on my laptop but after building the project it consumes lot of disk space then it's not working on port 8080 aftrer runing docker compose up.I removed all containers and everything now i have to clone repo by setting up ssh key with github.

**Comment by arnab15** [comment]
![error](https://user-images.githubusercontent.com/45819720/104770851-40933700-5797-11eb-8890-7f243228188a.png)

I don't know why these containers are not running.Is this project had any communication Chanel to seek help for setup?

**Comment by Yashs911** [comment]
@arnab15 https://github.com/internetarchive/openlibrary/blob/master/docker/README.md#fully-resetting-your-environment
you can ask your doubt on https://gitter.im/theopenlibrary/Lobby

**Comment by arnab15** [comment]
Thank you @Yashs911

**Comment by RayBB** [comment]
closing as it's unlikely we'll fix this directly but rather rewrite
